client/app/stories/index.jsx

```diff
@@ -22,6 +22,7 @@ import DropdownFillSmall from '../bundles/shared/components/Dropdown/DropdownFil
 import Footer from '../bundles/shared/components/Footer/Footer';
```

Comment:
When you rebase, this file should no longer exist. All the stories are now split up into separate files, so there should be one specifically for the MomentCards.